pkg/model/enum_def.go

```diff
@@ -56,3 +59,31 @@ func newEnumVal(orig string) EnumValue {
 		Clean:    string(clean),
 	}
 }
+
+// GetEnumDefs takes an *SDKAPI and *ackgenconfig.Config and returns a slice of
+// pointers to `EnumDef` structs which represent string fields whose value is
+// constrained to one or more specific string values.
+func GetEnumDefs(SDKAPI *SDKAPI, cfg *ackgenconfig.Config) ([]*EnumDef, error) {
```

Comment:
And this entirely new method?

Reply:
To keep the the `generate.go` as small as possible and allow other components to call `GetEnumDefs` without having to instantiate a `Generator/Inferrer`